src/app/users/users-profile/users-profile.scss

```diff
@@ -20,11 +20,17 @@
 
   .cards-container {
     display: flex;
+    flex-direction: row; 
+
+  @media (max-width: 576px) { 
```

Comment:
For consistency, you need to target a screen width that is defined in our `_variables.scss`

$screen-sm is a good option I believe